src/app/content/components/PrevNextBar.screenshotspec.tsx

```diff
@@ -0,0 +1,121 @@
+
+describe('PrevNextBar', () => {
+  it('renders correctly next element in mobile', async() => {
+    setMobileViewport(page);
+    await navigate(page, TEST_PAGE_URL);
+    await finishRender(page);
+    await scrollDown();
+    const screen = await page.screenshot();
+    expect(screen).toMatchImageSnapshot({
+      CI: {
+        failureThreshold: 1.5,
+        failureThresholdType: 'percent',
+      },
+    });
+
+  });
+
+  it('renders correctly next element on desktop', async() => {
+    setDesktopViewport(page);
+    await navigate(page, TEST_PAGE_URL);
+    await finishRender(page);
+    await scrollDown();
+    const screen = await page.screenshot();
+    expect(screen).toMatchImageSnapshot({
+      CI: {
+        failureThreshold: 1.5,
+        failureThresholdType: 'percent',
+      },
+    });
+  });
+
+  it('renders correctly prev and next elements on mobile', async() => {
+    setMobileViewport(page);
+    await navigate(page, TEST_PAGE_URL);
+    await finishRender(page);
+    await scrollDown();
+    expect(await clickNextLink(TEST_NEXT_PAGE_LINK, NEXT_PAGE_ARIA_LABEL)).toBe(true);
+    const screen = await page.screenshot();
+    expect(screen).toMatchImageSnapshot({
+      CI: {
+        failureThreshold: 1.5,
+        failureThresholdType: 'percent',
+      },
+    });
+  });
+
+  it('renders correctly prev and next elements on desktop', async() => {
+    setDesktopViewport(page);
+    await navigate(page, TEST_PAGE_URL);
+    await finishRender(page);
+    await scrollDown();
+    expect(await clickNextLink(TEST_NEXT_PAGE_LINK, NEXT_PAGE_ARIA_LABEL)).toBe(true);
+    const screen = await page.screenshot();
+    expect(screen).toMatchImageSnapshot({
+      CI: {
+        failureThreshold: 1.5,
+        failureThresholdType: 'percent',
+      },
+    });
+  });
+
+  it('renders correctly prev element on mobile', async() => {
+    setMobileViewport(page);
+    await navigate(page, TEST_LAST_PAGE_URL);
+    await finishRender(page);
+    await scrollDown();
+    const screen = await page.screenshot();
+    expect(screen).toMatchImageSnapshot({
+      CI: {
+        failureThreshold: 1.5,
+        failureThresholdType: 'percent',
+      },
+    });
+  });
+
+  it('renders correctly prev element on desktop', async() => {
+    setDesktopViewport(page);
+    await navigate(page, TEST_LAST_PAGE_URL);
+    await finishRender(page);
+    await scrollDown();
+    const screen = await page.screenshot();
+    expect(screen).toMatchImageSnapshot({
+      CI: {
+        failureThreshold: 1.5,
+        failureThresholdType: 'percent',
+      },
+    });
+  });
+
+});
+
+const scrollDown = () => page.evaluate(() => {
+  return window && document && document.documentElement && window.scrollBy(0, document.documentElement.scrollHeight);
+});
+
+// tslint:disable-next-line:no-shadowed-variable
+const clickNextLink = (href: string, elementToClick: string) => page.evaluate(async(href, elementToClick) => {
```

Comment:
[page.click](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pageclickselector-options) is a thing that could replace a bunch of the stuff in this function i think